### PR TITLE
regplot update to return OLS summary statistics

### PR DESCRIPTION
**Pull request by mklarqvist**
The returns OLS fit is very useful for retrieving correlations and other meaningful statistics

```
fit = regplot(y='area_legs',x='volume_legs',res_hue_limit=0.05,dataframe=df,destination='test.jpg')
fit.summary()
```

test

Screen Shot 2021-02-18 at 4 32 32 PM

### CODE REVIEW

File: ml4h/plots.py [hunk 1]
```diff
@@ -3242,24 +3252,40 @@ def regplot(
...
 
-    test = df[x].drop_duplicates().sort_values()
+    test = dataframe[x].drop_duplicates().sort_values()
     predictions = res.get_prediction(sm.add_constant(test))
     predictions = pd.concat([test, predictions.summary_frame(alpha=0.05)], axis=1)
     predictions.head()
 
-    residuals = pd.DataFrame({x: df[x], y: res.resid})
-    fig, axes = plt.subplots(1, 3, figsize=(20, 8))
+    # residuals = pd.DataFrame({x:dataframe[x],y:res.resid})
```
Comment (ndiamant) [inline ml4h/plots.py:3270]: left in on accident?

File: ml4h/plots.py [hunk 2]
```diff
@@ -3242,24 +3252,40 @@ def regplot(
         dataframe (pd.DataFrame): Pandas DataFrame with columns `x` and `y` and optionally `hue`
         hue (str, optional): Hue string name. Defaults to None.
         destination (str optional): Path to store output image. Defaults to None.
+        res_hue_limit (float, optional): Threshold of residuals deemed outliers. Defaults to None.
+        sizes (tuple, optional): Tuple of integers controlling point sizes. Defaults to (25, 25).
+        markers (tuple, optional): Tuple of chars controlling point types. Defalts to ["o", "o"].
```
Comment (ndiamant) [inline ml4h/plots.py:3257]: which point sizes, points and outliers?

**Review by ndiamant — CHANGES_REQUESTED** [review]
A few nitpicks then ready

**Comment by mklarqvist** [comment]
Good points. Updated

**Review by ndiamant — APPROVED** [review]

**Pull request merged**